tests/datavolume_test.go

```diff
@@ -693,6 +693,31 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 					Message: "Import Complete",
 					Reason:  "Completed",
 				}}),
+			table.Entry("[rfe_id:TBD][test_id:TBD]succeed creating import dv with given valid registry url and DV barely big enough", dataVolumeTestArguments{
+				name:             "dv-import-registry",
+				size:             "22Mi", // The image has 18M
+				url:              tinyCoreIsoRegistryURL,
+				dvFunc:           createRegistryImportDataVolume,
+				eventReason:      controller.ImportSucceeded,
+				phase:            cdiv1.Succeeded,
+				checkPermissions: true,
+				repeat:           10,
```

Comment:
Any particular reason we are repeating this 10 times? The original reason the 10 times was added for a registry import is that for hostpath WFFC storage we had a bug where there was a race that would only show every once in a while and running the test a bunch of times made it show up enough to debug it. But in this case I don't see a reason to repeat it 10 times.

Reply:
No particular reason, I meant "do the same, only with smaller PVC". A can remove the line.

Reply:
Done